### PR TITLE
[HOPSWORKS-2481] Move hops-system to a different recipe

### PR DESCRIPTION
**Pull request by javierdlrm**

### CODE REVIEW

**Comment by jimdowling** [comment]
If kfserving_enabled is set to 'false', should filebeat_enabled also be false?

File: recipes/addons.rb [hunk 1]
```diff
@@ -67,10 +67,9 @@
   node.override['kube-hops']['docker_img_reg_url'] = registry_host + ":#{node['hops']['docker']['registry']['port']}"
 end
 
+include_recipe "kube-hops::hops-system"
+include_recipe "kube-hops::filebeat"
+
 if node['kube-hops']['kfserving']['enabled'].casecmp?("true")
   include_recipe "kube-hops::kfserving"
```
Comment (jimdowling) [inline recipes/addons.rb:74]: Do we need an else statement here?

else
 node.override['kube-hops']['filebeat']['enabled'] = "false"
end

**Review by jimdowling — CHANGES_REQUESTED** [review]

File: recipes/addons.rb (hunk 1 above)
Comment (javierdlrm) [reply recipes/addons.rb:74]: Filebeat is independent from KFServing. It is also used to harvest logs from the standard Kubernetes deployments (our default serving tool). Thus, if Kubernetes is installed, filebeat needs to be installed as well. 

I removed the attribute _kube-hops/filebeat/enabled_.

Comment (javierdlrm) [reply recipes/addons.rb:74]: Another option is to make the installation of filebeat optional (via _kube-hops/filebeat/enabled_), but then if it's **false** server logs will not be collected when models are served on Kubernetes (either using normal deployments or kfserving)

**Review by jimdowling — APPROVED** [review]

**Pull request merged**